era_5g_relay_network_application/worker_service.py

```diff
@@ -63,12 +102,18 @@ def run(self) -> None:
             try:
                 sid, data = self.requests_queue.get(block=True, timeout=1)
```

Comment:
`except (Empty, Full):` could be after this line

Reply:
It could, but then there would have to be `continue` instead of `pass`. Since I'm not the original author of this except clause, I left it this way (I don't think that this matters much anyway)...